### DISCUSSION TITLE
Handle requests manually

### DISCUSSION THREAD

**Discussion by shark2k** [sct/overseerr#383] [General]
I realize this might seem strange for a system like this.
I've never used Ombi so not sure how this might be handled there and Petio, the other app like this that was announced around the same time, currently works like this.

I could be missing something and this might be possible now, but what I am looking for is to basically just receive an email when a new request is added.  I don't use Radarr or Sonarr or any other services like those.  I prefer to get my media manually as I have certain processes I do, that I can probably automate but don't feel like trying to do, just a personal choice.

I just set up Overseerr and when I made a test request I noticed that it said it would be approved right away, which I would rather not be the case.  I would prefer to have the request come in and then I would approve or deny it (with the ability to add the reason why I denied it).   Along with that, it would be nice if after approving there could be different status' such as in progress, adding soon and complete (maybe even make that user configurable where status' can be entered and then that becomes a running list of status but new ones can be added or selected from the current list).

This is a pretty slick app though and I appreciate all the work that you have put into this (and continue to put into it).

-Shark2k

**Comment by sct** [comment]
Okay so this is actually possible. You don't necessarily need to setup Radarr or Sonarr. The reason you are getting no notifications is because you are admin or the user has the auto approve permission. No notifications are sent for a request if it's auto approved.

If another user requests content, you should receive an email!

**Reply by shark2k** [reply]
(I swear I wasn't stalking this lol.  I was just about to go to bed and got the email notification :D).

Like I said, I figured I was missing something and what you said makes perfect sense.
Of course, as I said, I made the test request and since I am the admin all the permissions are checked.
Now that you mentioned it, I do recall seeing the auto approve permission (and couldn't remember where I saw that when I did the test request) and just verified that was where it was.

Appreciate the quick response on this as well as the easy going attitude (some people might not feel inclined to ask dumb questions like these for fear of possibly being ridiculed, so super appreciative of your "tone" in the response).

I'm definitely going to have to set up my reverse proxy for this so that I can get some of my users to jump on and test it out.

Awesome job again and thanks!

-Shark2k
